user_sync/app.py

```diff
@@ -75,6 +75,15 @@ def process_args():
     parser.add_argument('-d', '--remove-list',
                         help='specifies the file containing the list of users to be removed. Users on this list are removeFromOrg\'ed on the Adobe side.',
                         metavar='input_path', dest='remove_list_input_path')
+    parser.add_argument('--delete-nonexistent-users',
```

Comment:
ah yeah I thought I'd add it in a different submission, as I just wanted to make sure the change was headed in the right direction. I'll start on that after this pull request goes through if that's okay.